assets/js/components/InstanceOverview/InstanceOverview.jsx

```diff
@@ -17,14 +19,30 @@ function InstanceOverview({
     features,
     host_id: hostID,
     host,
+    absent_at: absentAt,
   },
 }) {
   const isDatabase = DATABASE_TYPE === instanceType;
-
+  const rowClasses = classNames(
+    { 'bg-gray-100': absentAt },
+    'table-row border-b'
+  );
   return (
-    <div className="table-row border-b">
-      <div className="table-cell p-2">
-        <HealthIcon health={health} />
+    <div className={rowClasses}>
+      <div className="table-cell p-2 px-5">
+        {absentAt ? (
+          <Tooltip content="Instance currently not registered." place="bottom">
```

Comment:
I would change text to `Instance current absent` or `Instance not found` (as in the mockup), because it is registered indeed